includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -666,6 +666,7 @@ public function generate_multiselect_html( $key, $data ) {
 			'desc_tip'          => false,
 			'description'       => '',
 			'custom_attributes' => array(),
+			'allbuttons'        => false,
```

Comment:
I agree - but I am re-using the same param that's been used in some extensions to achieve this, so that it's a consistent and seamless transition for those extensions.